src-frontend/app/templates/components/navigation-bar.hbs

```diff
@@ -0,0 +1,24 @@
+<nav class="navbar navbar-default">
+    <div class="container-fluid">
+        <div class="navbar-header">
+            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+                <span class="sr-only">Toggle navigation</span>
+                <span class="icon-bar"></span>
```

Comment:
Can't we use something like `glyphicon glyphicon-menu-hamburger` 
![screen shot 2015-06-29 at 4 44 56 pm](https://cloud.githubusercontent.com/assets/978947/8418028/6a901f94-1e7e-11e5-937f-2df4be11a8fe.png)